File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        
        numDict = {}
        sortedDict = []
        result = []

        for num in nums:
            numDict[num] = 1 + numDict.get(num, 0)

        for key in numDict:
            sortedDict.append([numDict[key], key])
        
        sortedDict.sort(reverse = True)

        for i in range(k):
            result.append(sortedDict[i][1])
   

        return result

        

File: Data Structures & Algorithms/squares-of-a-sorted-array/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number[]}
     */
    sortedSquares(nums) {

        let temp = []
        for(let num of nums){
            temp.push(num*num)
        }
        return temp.sort((a,b) => a - b)
    }
}
